### DISCUSSION TITLE
Add another html page to current timline

### DISCUSSION THREAD

**Discussion by scarlettwwx** [jspsych/jsPsych#1093] [Q&A]
Hello,

I currently have a well-functionalized separate HTML page to perform the headphone check. How should I add this step during the experiment? I tried to use external-html to call it, however, it cannot show properly (Since this plugin is only for simple documents like a consent form?). Does anyone have an idea to fix this? Thanks in advance!!!

**Comment by becky-gilbert** [comment]
Hi @scarlettwwx, you're right that the external-html plugin is mostly for showing a non-jsPsych HTML page in the middle of a jsPsych experiment, so I don't think that's what you want to use here. 

How are you hosting your experiment? Most hosting platforms provide a way of linking together multiple pages, one after another, where each page contains one jsPsych 'experiment' (really this might be just one part of your larger experiment, like consent, headphone check, main task, etc.).

**Reply by scarlettwwx** [reply]
Hi! I just use Github Pages to hold the experiment

**Comment by jodeleeuw** [comment]
Hi @scarlettwwx 

My normal approach would be to convert the non-jsPsych page into a plugin. Once you have a functional HTML page it usually isn't too complicated to make the conversion. (We've also been eager to add a headphone check plugin to the library, so it could be a nice contribution too 😄.)

Alternatively, you could just direct folks to the headphone check page first and then use that to link to your experiment.
